### PR TITLE
Bug 1599522 - Change error statements when discovery fails

### PR DESCRIPTION
**Pull request by dymurray**
Display warning when discovery fails because user can still specify images in config.

### CODE REVIEW

**Review by djzager — APPROVED** [review]

File: registries/adapters/apiv2_adapter.go [hunk 1]
```diff
@@ -120,7 +120,8 @@ func (r APIV2Adapter) GetImageNames() ([]string, error) {
 	// discover images from URL
 	discoveredImages, err := r.discoverImages(fmt.Sprintf(apiV2CatalogURL, r.config.URL))
 	if err != nil && len(imageList) == 0 {
-		return nil, err
+		// This means no images specified in config and discovered no images. Erroring out
+		return nil, errors.New("failed to fetch catalog response: [%v] and failed to find images in configuration")
```
Comment (eriknelson) [inline registries/adapters/apiv2_adapter.go]: This is missing the argument to the format string (figure it's just err?)

**Review by eriknelson — CHANGES_REQUESTED** [review]

**Review by jmrodri — CHANGES_REQUESTED** [review]
NACK, what @eriknelson pointed out. Also, the PR description is really long. Whoever commits it please rephrase it to something shorter.

**Comment by dymurray** [comment]
Thanks @eriknelson and @jmrodri. updated.

**Review by jmrodri — APPROVED** [review]

**Comment by jmrodri** [comment]
@eriknelson issue was addressed by @dymurray so I'm merging this.
